### PR TITLE
Refactor models into new files to make them more managable

### PR DESCRIPTION
**Pull request by MaxPresman**
hey CFA,

was browsing the project and noticed that the models are taking the majority of the app.py... so I refactored them into separate files to keep them manageable.

please give it some :eyes:

### CODE REVIEW

File: models/organization.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+from . import db
+from flask import request
+from sqlalchemy import desc, DDL, event
+from column_types import TSVectorType
+from . import Project, Event, Story
+from datetime import date, datetime
+import time
+
+
+def safe_name(name):
```
Comment (MaxPresman) [inline models/organization.py]: need to refactor this to use the module, one moments.

**Comment by MaxPresman** [comment]
I gave app.py one more step by moving the routes into blueprints and reducing ~100 LOC.. https://github.com/MaxPresman/cfapi/compare/refactor_models...MaxPresman:blueprint_project 

i'll open another PR to not change too much stuff together.

**Comment by ondrae** [comment]
THANK YOU! I'll review this on Tuesday but it looks good.

**Comment by ondrae** [comment]
Hi @MaxPresman. I really like how you've split up the models and utils into something much cleaner.

I've got your code up and running on our staging server at https://cfapi-staging.herokuapp.com/api/organizations

The one problem is with `run_update.py`. It is supposed to run once an hour to pull in all the fresh new activity across the network. I'm getting this error when I try and run it.

```
File "run_update.py", line 1078, in <module>
    main(org_name=org_name, org_sources=args.org_sources)
  File "run_update.py", line 969, in main
    existing_org = db.session.query(Organization).filter(filter).first()
...
RuntimeError: application not registered on db instance and no application bound to current context
```

**Comment by MaxPresman** [comment]
ah, good catch, I was not familiar with the run_updater, i'll fix it up

**Comment by MaxPresman** [comment]
i updated the file to init the db context; it should work, but I am curious why the testing suite did not catch it.

**Comment by MaxPresman** [comment]
going to close for a moment.

**Pull request closed without merging**